src/core/binformat.js

```diff
@@ -396,7 +422,19 @@ exports.ledger = {
     ['LedgerIndex',         OPTIONAL],
     ['Balance',             REQUIRED],
     ['LowLimit',            REQUIRED],
-    ['HighLimit',           REQUIRED]])
+    ['HighLimit',           REQUIRED]]),
+  SusPay: [117].concat(sleBase,[
```

Comment:
I think this probably wants to be `SuspendedPayment`, as `SusPay` is just an abbreviation for internals

@vinniefalco ?